app/views/finance/participants/_ecf_profile.html.erb

```diff
@@ -38,11 +38,7 @@
   <% summary_list.row do |row| %>
     <% row.key { "Training status" } %>
     <% row.value { pp.training_status } %>
```

Comment:
I think we need to remove surfacing this value as well 🤔 ?

Reply:
Addressed tks @cwrw